components/support/migration/docs/metrics.md

```diff
@@ -24,59 +24,70 @@ The following metrics are added to the ping:
 | migration.fxa.has_custom_idp_server |[boolean](https://mozilla.github.io/glean/book/user/metrics/boolean.html) |Is custom idp server configured?  |[1](https://github.com/mozilla-mobile/android-components/pull/5582#issuecomment-574673110)||2020-10-01 |
 | migration.fxa.has_custom_token_server |[boolean](https://mozilla.github.io/glean/book/user/metrics/boolean.html) |Is custom token server configured?  |[1](https://github.com/mozilla-mobile/android-components/pull/5582#issuecomment-574673110)||2020-10-01 |
 | migration.fxa.success_reason |[counter](https://mozilla.github.io/glean/book/user/metrics/counter.html) |Why this migration succeeded. See codes in TelemetryHelpers.kt  |[1](https://github.com/mozilla-mobile/android-components/pull/5483#issuecomment-573971458)||2020-10-01 |
+| migration.fxa.total_duration |[timespan](https://mozilla.github.io/glean/book/user/metrics/timespan.html) |Total duration of this migration.  |[1](TODO)||2020-10-01 |
 | migration.fxa.unsupported_account_version |[string](https://mozilla.github.io/glean/book/user/metrics/string.html) |What is the detected unsupported account version?  |[1](https://github.com/mozilla-mobile/android-components/pull/5582#issuecomment-574673110)||2020-10-01 |
 | migration.fxa.unsupported_pickle_version |[string](https://mozilla.github.io/glean/book/user/metrics/string.html) |What is the detected unsupported pickle version?  |[1](https://github.com/mozilla-mobile/android-components/pull/5582#issuecomment-574673110)||2020-10-01 |
 | migration.fxa.unsupported_state_version |[string](https://mozilla.github.io/glean/book/user/metrics/string.html) |What is the detected unsupported state version?  |[1](https://github.com/mozilla-mobile/android-components/pull/5582#issuecomment-574673110)||2020-10-01 |
 | migration.gecko.any_failures |[boolean](https://mozilla.github.io/glean/book/user/metrics/boolean.html) |Did this migration encounter any failures (exceptions)?  |[1](https://github.com/mozilla-mobile/android-components/pull/5483#issuecomment-573971458)||2020-10-01 |
 | migration.gecko.failure_reason |[counter](https://mozilla.github.io/glean/book/user/metrics/counter.html) |Why this migration failed. See codes in TelemetryHelpers.kt  |[1](https://github.com/mozilla-mobile/android-components/pull/5655#pullrequestreview-346359792)||2020-10-01 |
 | migration.gecko.success_reason |[counter](https://mozilla.github.io/glean/book/user/metrics/counter.html) |Why this migration succeeded. See codes in TelemetryHelpers.kt  |[1](https://github.com/mozilla-mobile/android-components/pull/5655#pullrequestreview-346359792)||2020-10-01 |
+| migration.gecko.total_duration |[timespan](https://mozilla.github.io/glean/book/user/metrics/timespan.html) |Total duration of this migration.  |[1](TODO)||2020-10-01 |
```

Comment:
This is the only one that I'm not entirely clear on - what does it mean to "migrate Gecko". Gecko prefs? Gecko dbs? additional Gecko-y data?

Reply:
From the migration code comments:
```
        // GeckoView will happily pick up the profile from Fennec and reuse all data in it. So this
        // migration is mostly focused on removing the files that we do not want to reuse.

        // The prefs file contains gecko prefs that have been changed by the user (about:config) or
        // code. Since those may be incompatible with GeckoView and/or app code, we decided to just
        // reset them. GeckoView will recreate the file with defaults if needed.
```